sdk/tests/conformance/textures/misc/mipmap-fbo.html

```diff
@@ -117,9 +118,28 @@
                         "Framebuffer with a non-mipmap complete cube map attachment should be green");
 }
 
+var testIncompleteMipmapInFBO = function () {
```

Comment:
After discussion with @zhenyao and @toji we agree that tests of new WebGL 2.0 functionality should not be added to the conformance/ directory using tests of getDefault3DContextVersion. Tests of new functionality should be put into conformance2/ .

In order to save time I'm going to merge this as is, but am going to refactor it immediately afterward to move the new test. Please follow this example in the future. Thanks.